src/main/java/org/opendevstack/provision/controller/ProjectApiController.java

```diff
@@ -68,7 +68,7 @@
   private static final String STR_LOGFILE_KEY = "loggerFileName";
   
   @Autowired
-  private JiraAdapter jiraAdapter;
+  JiraAdapter jiraAdapter;
```

Comment:
Why was `private` removed?

Also further down, visibility is mixed between private and package. Can we use one visibility level?